File: ScantronPDFProducer/libs/PDFjet.78/Example_05.java
import java.lang.*;
import java.io.*;

import com.pdfjet.*;


/**
 *  Example_05.java
 *
 */
public class Example_05 {

    public Example_05() throws Exception {

        FileOutputStream fos = new FileOutputStream("Example_05.pdf");

        PDF pdf = new PDF(fos);

        Font f1 = new Font(pdf, "Helvetica");

        Page page = new Page(pdf, Letter.PORTRAIT);

        TextLine text = new TextLine(f1);
        text.setPosition(300.0, 300.0);
        for (int i = 0; i < 360; i += 15) {
            text.setTextDirection(i);
            text.setUnderline(true);
            // text.setStrikeLine(true);
            text.setText("             Hello, World -- " + i + " degrees.");
            text.drawOn(page);
        }

        text = new TextLine(f1, "WAVE AWAY");
        text.setPosition(70.0, 50.0);
        text.drawOn(page);

        f1.setKernPairs(true);
        text.setPosition(70.0, 70.0);
        text.drawOn(page);

        f1.setKernPairs(false);
        text.setPosition(70.0, 90.0);
        text.drawOn(page);

        f1.setSize(8);
        text = new TextLine(f1, "-- font.setKernPairs(false);");
        text.setPosition(150.0, 50.0);
        text.drawOn(page);
        text.setPosition(150.0, 90.0);
        text.drawOn(page);
        text = new TextLine(f1, "-- font.setKernPairs(true);");
        text.setPosition(150.0, 70.0);
        text.drawOn(page);

        Point point = new Point(300.0, 300.0);
        point.setShape(Point.CIRCLE);
        point.setFillShape(true);
        point.setColor(RGB.BLUE);
        point.setRadius(37.0);
        point.drawOn(page);
        point.setRadius(25.0);
        point.setColor(RGB.WHITE);
        point.drawOn(page);

        pdf.flush();
        fos.close();
    }


    public static void main(String[] args) {
        try {
            new Example_05();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

}   // End of Example_05.java

File: ScantronPDFProducer/libs/PDFjet.78/Example_05.cs
using System;
using System.IO;

using PDFjet.NET;


/**
 *  Example_05.cs
 *
 */
public class Example_05 {

    public Example_05() {

        FileStream fos = new FileStream("Example_05.pdf", FileMode.Create);
        BufferedStream bos = new BufferedStream(fos);

        PDF pdf = new PDF(bos);
        pdf.setCompressor(Compressor.ORIGINAL_ZLIB);

        // Before you enable this flag please read README.ZLIB.TXT
        // in the 'optional' directory.

        // If PDF/A is not required use Helvetica, TimesRoman or Courier
        Font f1 = new Font(pdf, "Helvetica");

        Page page = new Page(pdf, Letter.PORTRAIT);

        TextLine text = new TextLine(f1);
        text.SetPosition(300.0, 300.0);
        for (int i = 0; i < 360; i += 15) {
            text.SetTextDirection(i);
            text.SetUnderline(true);
            // text.SetStrikeLine(true);
            text.SetText("             Hello, World -- " + i + " degrees.");
            text.DrawOn(page);
        }

        text = new TextLine(f1, "WAVE AWAY");
        text.SetPosition(70.0, 50.0);
        text.DrawOn(page);

        f1.SetKernPairs(true);
        text.SetPosition(70.0, 70.0);
        text.DrawOn(page);

        f1.SetKernPairs(false);
        text.SetPosition(70.0, 90.0);
        text.DrawOn(page);

        f1.SetSize(8);
        text = new TextLine(f1, "-- font.SetKernPairs(false);");
        text.SetPosition(150.0, 50.0);
        text.DrawOn(page);
        text.SetPosition(150.0, 90.0);
        text.DrawOn(page);
        text = new TextLine(f1, "-- font.SetKernPairs(true);");
        text.SetPosition(150.0, 70.0);
        text.DrawOn(page);

        Point point = new Point(300.0, 300.0);
        point.SetShape(Point.CIRCLE);
        point.SetFillShape(true);
        point.SetColor(RGB.BLUE);
        point.SetRadius(37.0);
        point.DrawOn(page);
        point.SetRadius(25.0);
        point.SetColor(RGB.WHITE);
        point.DrawOn(page);

        pdf.Flush();
        bos.Close();
    }


    public static void Main(String[] args) {
        try {
            new Example_05();
        } catch (Exception e) {
            Console.WriteLine(e.StackTrace);
        }
    }

}   // End of Example_05.cs
